pact.cabal

```diff
@@ -454,3 +455,5 @@ test-suite hspec
         , servant-client
         , temporary >= 1.3
         , yaml
+        , process
```

Comment:
It looks like we removed all use of `process`, maybe we can delete this dependency?